demo/card.html

```diff
@@ -0,0 +1,167 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+		<title>cosmoz-tab-card demo</title>
+
+		<script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+
+		<link rel="import" href="../../iron-demo-helpers/demo-pages-shared-styles.html" />
+		<link rel="import" href="../../iron-demo-helpers/demo-snippet.html" />
+
+		<link rel="import" href="../cosmoz-tab-card.html" />
+		<link rel="import" href="../cosmoz-tab.html" />
+		<link rel="import" href="../cosmoz-tabs.html" />
+
+		<custom-style>
+			<style is="custom-style" include="demo-pages-shared-styles">
+				#container {
+					 max-width: 800px;
+				 }
+			</style>
+		</custom-style>
+	</head>
+	<body unresolved>
+		<div id="container" class="vertical-section-container centered">
+			<h3>Plain <code>cosmoz-tabs</code> with <code>cosmoz-tab-card</code></h3>
+			<demo-snippet>
+				<template>
+					<cosmoz-tabs>
+						<cosmoz-tab heading="Tab1" tab-id="tab1">
+							<cosmoz-tab-card>
```

Comment:
Could you just add `heading` attribute to all the tab-card,  like you did for the accordion mode ?
That way, one can see how it renders in accordion and not accordion.